### DISCUSSION TITLE
xts mode

### DISCUSSION THREAD

**Discussion by sidielnikov** [HMBSbige/CryptoBase#64] [Ideas]
Bro, please add XTS mode for AES

Here are a few examples to help you

https://github.com/tkp1n/ArmAes    
not working decryption. If it's not difficult to tell me how to fix the function 
DecryptDataUnit256(
        in AesKey key,
        ref byte tweak,
        ref byte ct, nuint ctLen,
        ref byte pt)

https://github.com/mikhail-khalizev/XTSSharp

**Comment by HMBSbige** [comment]
Support was added in version [2.0.0](https://github.com/HMBSbige/CryptoBase/releases/tag/2.0.0)
see: https://github.com/HMBSbige/CryptoBase/wiki/XTS
